app/config.go

```diff
@@ -175,7 +176,8 @@ func InitConfig(datadir, tmNode, persistentPeers, seeds, tmRPCPort, tmPeersPort
 		// ensure directory path made
 		err = os.MkdirAll(c.PocketConfig.DataDir+FS+ConfigDirName, os.ModePerm)
 		if err != nil {
-			panic(err)
+			fmt.Println(err)
+			os.Exit(1)
```

Comment:
```suggestion
			log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work